src/qutip_jax/binops.py

```diff
@@ -74,7 +74,7 @@ def mul_jaxarray(matrix, value):
     # We don't want to check values type in case jax pass a tracer etc.
     # But we want to ensure the output is a matrix, thus don't use the
     # fast constructor.
-    return JaxArray(matrix._jxa * value)
```

Comment:
May I ask why the shape is necessary here but not in the other uses of `_fast_constructor`?